### PR TITLE
https://github.com/eclipse/xtext/issues/1862 Guava 30.1.0

### PR DESCRIPTION
**Pull request by miklossy**
Update Google Guava version from 27.1.0 to 30.1.0

Signed-off-by: miklossy <miklossy@itemis.de>

### CODE REVIEW

**Comment by cdietrich** [comment]
miss change to target file

File: org.eclipse.xtext.dev-bom/build.gradle [hunk 1]
```diff
@@ -9,9 +9,9 @@ dependencies {
         api "args4j:args4j:2.33"
         api "com.google.code.findbugs:jsr305:3.0.2"
         api "com.google.errorprone:error_prone_annotations:2.2.0"
```
Comment (cdietrich) [inline org.eclipse.xtext.dev-bom/build.gradle:11]: i assume this one needs to be changed too

Comment (cdietrich) [reply org.eclipse.xtext.dev-bom/build.gradle:11]: @miklossy this is still open

**Comment by miklossy** [comment]
> miss change to target file

@cdietrich I have just updated the Guava version in the `xtext-lib.target.target` file

File: org.eclipse.xtext.dev-bom/build.gradle (hunk 1 above)
Comment (miklossy) [reply org.eclipse.xtext.dev-bom/build.gradle]: @cdietrich I have just updated the version of `com.google.errorprone:error_prone_annotations` to `2.3.4`, according to
screenshot

**Review by cdietrich — APPROVED** [review]

**Pull request merged**